config.js

```diff
@@ -1,12 +1,16 @@
 const mongoose = require("mongoose");
+const dotenv = require ('dotenv');
 
+dotenv.config();
 
- mongoose.connect("mongodb://localhost:27017/inventory-mgt",  {
+
+ mongoose.connect(process.env.MONGO_URL, {
```

Comment:
```suggestion
 mongoose.connect(process.env.DB_URL, {
```
please change the variable name to be more descriptive for example DB_URL.